Sources/Commands/SwiftTool.swift

```diff
@@ -382,6 +375,18 @@ public class SwiftTool<Options: ToolOptions> {
             var options = Options()
             try binder.fill(parseResult: result, into: &options)
 
+            verbosity = Verbosity(rawValue: options.verbosity)
+
+            buildingProgressAnimation = NinjaProgressAnimation(
+                stream: stdoutStream,
+                isVerbose: verbosity != .concise)
```

Comment:
IMO we shouldn't be using the Ninja progress animation when we're verbose. However, @hartbit has more context about what the right thing is here.

Reply:
The correct fix is the modify the BuildDelegate to bypass the `ProgressAnimation` when verbose. I was going to write a fix, if that's okay.

Reply:
Sounds good! Thanks @hartbit! I’ll close this now.